.flake8

```diff
@@ -39,6 +39,28 @@ max-complexity = 10
 # Accessibility/large fonts and PEP8 friendly:
 max-line-length = 79
 
+# https://wemake-python-stylegui.de/en/latest/pages/usage/formatter.html
+format = wemake
+
+# wemake-python-styleguide
+show-source = true
+
+# Allow certain violations in certain files:
+per-file-ignores =
+
+  # E800 reports a lot of false-positives for legit
```

Comment:
is this still true?

Reply:
Yep, it's complaining about `# isort: split` legit comment.